.travis.yml

```diff
@@ -10,10 +11,13 @@ script:
 - tox
 - coverage run --source isatools -m py.test
 - coverage report -m
-branches:
-  only:
-    - master
-    - develop
+- tests/my-pip-uninstall matplotlib pandas numpy tox-travis
+- rm -rf .tox
+- PYTHONPATH="${PWD}" python tests/test_isatab_only_meta.py
+#branches:
```

Comment:
To test the partial functionality without deps, we first run the regular full tests, then uninstall matplotlib, pandas, numpy, tox-travis and re-run only the metadata parsing tests.

Uninstalling the packages proved to be annoying.  `pip uninstall` left empty package directories when testing with Python 3.  I had to resort to custom little uninstallation script which deletes the directories after running `pip uninstall`.